docs/customizations.md

```diff
@@ -247,12 +247,10 @@ class ServiceClient(ServiceClientGenerated):
 __all__ = ["ServiceClient"]
 ```
 
-## Postprocessing
+## Postprocessing (Optional)
 
-All of the code, linting, and intellisense between generated and handwritten code should work immediately. In some cases, to get mypy clean, you will need to run our postprocessing step. To run this step, run
+All of the code, linting, and intellisense between generated and handwritten code should work immediately. In some cases, to fix import issue, you will need to run our postprocessing step. To run this step, run
```

Comment:
postprocessing is actually to get the generated code mypy clean, so I wouldn't change this line. It gets the code mypy clean by updating the imports in the `__init__.py` to explicitly import from `_patch.py` because otherwise `mypy` gets mad, there is no actual import issue.

I'm ok to say that it's optional though on line 250, but I think 252 should stay the same

Reply:
@iscai-msft `get mypy clean` may give users illusion that postprocess can help fix all the mypy issues. How about `fix import issue to pass mypy check`? It shall provide moderate expectation for users

Reply:
Fine.
Updated.